packages/utils-query-operation/test/FragmentSelectorShapes-test.ts

```diff
@@ -338,5 +337,105 @@ describe('FragmentSelectorShapes', () => {
         undefined!,
       ))).toBeTruthy();
     });
+
+    describe('with extension functions', () => {
+      let extensionFunctionExpression: Algebra.NamedExpression;
+
+      beforeAll(() => {
+        extensionFunctionExpression = {
```

Comment:
Could you also add one (or more) test(s) where the expression is contained within some other operation?